packages/one-shot/paas-1962-apply-8032-patch.yml

```
@@ -0,0 +1,53 @@
+---
+type: update
+version: 1.5.2
+name: PAAS-1962 - Apply 8.0.3.2 patch to 8.0.3 envs
+id: paas-1962-apply-8032-patch
+
+mixins:
+  - "../../mixins/common.yml"
+  - "../../mixins/jahia.yml"
+
+onInstall:
+  - checkJahiaVersion
+  - checkPatchAlreadyInstalled
+  - installPatch
+  - checkPatchInstallation
+
+actions:
+  checkJahiaVersion:
+    - getJahiaVersion
+    - if (${globals.jahiaVersion} != "8.0.3.0"):
+        - return:
+            type: success
+            message: "Jahia version is ${globals.jahiaVersion}, not 8.0.3.0 ; aborting"
```

```suggestion
            message: "Jahia version is ${globals.jahiaVersion}, not 8.0.3.0 ; nothing to do"
```
for the record :trollface:

---

Number of fucks given : :zero: